src/Nethermind/Nethermind.Synchronization/SnapSync/ProgressTracker.cs

```diff
@@ -87,13 +87,13 @@ public void UpdatePivot()
                 int queueLength = AccountsToRefresh.Count;
                 AccountWithStorageStartingHash[] paths = new AccountWithStorageStartingHash[queueLength];
 
+                Interlocked.Increment(ref _activeAccRefreshRequests);
```

Comment:
I get that those increments should be first thing in any branch of this if, can we put it on top of every branch, even before `LogRequest` ect? To have better visibility and consistency of concurrency?

Reply:
Ok